common/src/Api.ts

```diff
@@ -78,3 +78,11 @@ export interface ApiMessagePayload {
 }
 
 export type FetchMessagesResponse = ApiResponse<ApiMessagePayload[]>;
+
+export interface DeleteRoomResponse {
```

Comment:
Very small detail, but I suggest using helper ApiResponse<{ message: string}> or something like that, It's no big deal but we have conflict here as well, so maybe just resolve conflict and correct it in one go